helix-view/src/document.rs

```diff
@@ -112,7 +112,7 @@ pub struct Document {
     // be more troublesome.
     pub history: Cell<History>,
 
-    pub savepoint: Option<Transaction>,
+    pub savepoint: Option<(i32, Transaction)>,
```

Comment:
Why are we using `i32` for version instead of `u32`?

Reply:
The document  version type is `i32`.